whylog/tests/test_constraints/__init__.py

```diff
@@ -41,20 +36,6 @@ def setUp(self):
         self.min_delta = timedelta(seconds=1)
         self.max_delta = timedelta(seconds=10)
 
-    def test_constructor_success(self):
```

Comment:
Now MAX_DELTA is not tested once. All this test class doesn't call `.verify()` even once and has zero asserts for functionality!